src/Telnyx.net/Services/Wireless/WDRReports/WDRReportsService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Telnyx.net.Entities;
+using Telnyx.net.Entities.Wireless.WDRReports;
+
+namespace Telnyx.net.Services.Wireless.WDRReports
+{
+    public class WDRReportsService : Service<WDRReportsRecord>,
+        IListable<WDRReportsRecord, ListWDRReportsOptions>,
+        ICreatable<WDRReportsRecord, CreateWDRReportsOptions>,
+        IRetrievable<WDRReportsRecord>,
+        IDeletable<WDRReportsRecord>
+    {
+        public override string BasePath => throw new NotImplementedException();
```

Comment:
need to add the endpoint here. I believe its "wireless/detail_records_reports"

Reply:
done.